T6_1/IRepository.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Homewokr_6_1
+{
```

Comment:
Better names are Load and Save (because its a generic interface, you can store anything, not only catalogs)